plugins/ruby-foreman-hdm/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: ruby-foreman-hdm
+Section: ruby
+Priority: optional
+Maintainer: Tim <tim@bastelfreak.de>
+Build-Depends: debhelper-compat (= 13),
+               gem2deb (>= 1),
+               rake,
+               foreman-assets (>= 3.4.0~rc1),
```

Comment:
aren't foreman-assets and foreman-nulldb a thing anymore? or do they only exist for develop?

Reply:
https://github.com/theforeman/foreman-packaging/pull/12067#issuecomment-2897449713